### DISCUSSION TITLE
Share Options

### DISCUSSION THREAD

**Discussion by writingpoli** [knowledgefutures/pubpub#1432] [Ideas]
Just looking at Gabe, et al.'s latest pub and it occurred to me that, as a reader, I'd like these share options added for easy access:

![Capture+_2021-05-28-08-37-31-01](https://user-images.githubusercontent.com/3883763/119993103-d4201400-bf90-11eb-9022-58181bcda641.jpeg)
![Capture+_2021-05-28-08-37-40-01](https://user-images.githubusercontent.com/3883763/119993120-d8e4c800-bf90-11eb-866e-48f8aa4ad18a.jpeg)

**Comment by jeffpooley** [comment]
+1 on both of these sharing options. (The KFG’s [article](https://commonplace.knowledgefutures.org/pub/kp81ylos/), by the way, is excellent and worth a read—in part for what it says about the philosophy behind PubPub.)
